src/main/java/duke/DateParser.java

```diff
@@ -0,0 +1,24 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Parses Date from String
+ */
+public class DateParser {
+    /**
+     * Takes the date string and parses it into LocalDate object
+     * @param date
+     * @return LocalDate object for the date string
+     */
```

Comment:
Should there be an empty line between description and parameter section?

```suggestion
    /**
     * Takes the date string and parses it into LocalDate object
     *
     * @param date
     * @return LocalDate object for the date string
     */
```